docker-entrypoint.sh

```diff
@@ -85,10 +85,18 @@ then
     error_with_msg "MariaDB did not stop"
 fi
 
-echo "- Generating JWT key"
-dd if=/dev/urandom bs=32 count=1 >/etc/slurm/jwt/jwt_hs256.key
-chown slurm:slurm /etc/slurm/jwt/jwt_hs256.key
-chmod 0600 /etc/slurm/jwt/jwt_hs256.key
+jwt_secret_dir="/etc/slurm/jwt/jwt_hs256.key"
```

Comment:
Incredibly minor but wouldn't `jwt_secret_file` or `jwt_secret_path` would be better than `jwt_secret_dir` as it's the full path to the file rather than to the directory